src/scripts/google-analytics.js

```diff
@@ -1,46 +1,75 @@
-/* istanbul ignore file */
-/* global ga */
-"use strict"
-
-module.exports.enableGoogleAnalytics = function enableGoogleAnalytics(gaUID) {
-  // if set to true disables google analytics
-  window[`ga-disable-${gaUID}`] = false
-
-  window.ga =
-    window.ga ||
-    function() {
-      ;(ga.q = ga.q || []).push(arguments)
-    }
-  ga.l = +new Date()
-  module.exports.anonymize()
-  ga(`create`, gaUID, `auto`)
-}
-
-module.exports.anonymize = function anonymize(anonymize = true) {
-  if (window.ga) {
-    ga(`set`, `allowAdFeatures`, !anonymize)
-    ga(`set`, `anonymizeIp`, anonymize)
-  }
-}
-
-module.exports.deanonymize = function deanonymize() {
-  module.exports.anonymize(false)
-}
-
-module.exports.disableGoogleAnalytics = function disableGoogleAnalytics(gaUID) {
-  // if set to true disables google analytics
-  window[`ga-disable-${gaUID}`] = true
-}
-
-module.exports.track = function track(...args) {
-  if (window.ga) {
-    window.ga(`send`, ...args)
-  }
-}
-
-module.exports.setGoogleAnalyticsPage = function track(...args) {
-  if (window.ga) {
-    window.ga(`set`, `page`, ...args)
-    window.ga(`send`, `pageview`)
-  }
-}
+/* istanbul ignore file */
+/* global ga */
+"use strict"
+
+module.exports.enableGoogleAnalytics = function enableGoogleAnalytics(gaUID) {
+  // if set to true disables google analytics
+  window[`ga-disable-${gaUID}`] = false
+
+  window.ga =
+    window.ga ||
+    function() {
+      ;(ga.q = ga.q || []).push(arguments)
+    }
+  ga.l = +new Date()
+  module.exports.anonymize()
+  ga(`create`, gaUID, `auto`)
+}
+
+module.exports.anonymize = function anonymize(anonymize = true) {
+  if (window.ga) {
+    ga(`set`, `allowAdFeatures`, !anonymize)
+    ga(`set`, `anonymizeIp`, anonymize)
+  }
+}
+
+module.exports.deanonymize = function deanonymize() {
+  module.exports.anonymize(false)
+}
+
+module.exports.disableGoogleAnalytics = function disableGoogleAnalytics(gaUID) {
+  // if set to true disables google analytics
+  window[`ga-disable-${gaUID}`] = true
+}
+
+function customToNum(custom) {

```

Comment:
what does this do?

Reply:
GA takes parameters like dimension1 or dimension2. It's not so clear to understand in code, that's why I decided to replace them with more reliable words like `address` and `network`, so we can pass parameter like address = 'sdkjashkd' and not dimension1 = ' sdkjashkd'. This code replaces those 'friendly' parameters to google once.

Reply:
comment this!

Reply:
```suggestion
/*
GA takes parameters like dimension1 or dimension2. This replaces the human readable values with the by GA required values.
*/
function customToNum(custom) {
```